### ISSUE TITLE
Abnormal fetal femur length

### ISSUE THREAD

**Issue by michellejoywang** [obophenotype/human-phenotype-ontology#11335]
**Preferred term label:**
Abnormal fetal femur length

**Synonyms**
Abnormal FL, abnormal fetal FL

**Definition (free text, please give PubMed ID)**
Femur length is measured between the outer border of one end of the the femoral diaphysis to the other outer border, excluding the epiphysis or other artifact. Abnormal femur length would be defined as less than 3%ile or 2 standard deviations below the mean. 

**Parent term (use hpo.jax.org/app)**
Aplasia/hypoplasia of the femur 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
100800, 187601, 166210 

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-5177-8109

**Comment by michellejoywang** [comment]
@VarenyaJ plz add PMID for Definition

**Comment by michellejoywang** [comment]
Short fetal femur length as child term of this HPO term

**Comment by pnrobinson** [comment]
@michellejoywang 
Slightly changing the definition as follows:

Adding new term: 
- ID: HP:6001418
- label: Abnormal fetal femur length
- parents: [Abnormal fetal skeletal morphology (HP:0025662)](https://hpo.jax.org/app/browse/term/HP:0025662)
- pmid: 
- def: Femur length is measured between the outer border of one end of the the femoral diaphysis to the other outer border, excluding the epiphysis or other artifact. Abnormal femur length is defined as being more than two standard deviations away from the mean.
- comment: 
- orcid: orcid:[0000-0002-5177-8109](https://orcid.org/0000-0002-5177-8109)
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11335
- synonyms:
